CryptoCore/Controllers/HomeController.cs

```diff
@@ -35,9 +35,19 @@ public async Task<IActionResult> Index()
             return View(model);
         }
 
-        public IActionResult Privacy()
+        public async Task<IActionResult> Privacy()
         {
-            return View();
+            var model = new TickerViewModel();
+            var response = await _binanceClient.GetTwentyFourHourTickerInfo();
+            foreach (var coin in response)
+            {
+                var tempObject = new TickerModel();
+                tempObject.Symbol = coin.symbol;
+                tempObject.PriceChangePercent = coin.priceChangePercent;
+                model.Coins.Add(tempObject);
+
+            }
+            return View(model);
```

Comment:
```suggestion
            return View();
```

Reply:
Keep the controllers clean until we need to send the data by. Less merge conflicts